src/@utils/nft.ts

```diff
@@ -57,16 +54,18 @@ export function generateNftMetadata(): NftMetadata {
 }
 
 export function generateNftCreateData(nftMetadata: NftMetadata): any {
+  // TODO: figure out if Buffer.from method is working in browser in final build
+  // as BTOA is deprecated.
+  // tokenURI: window?.btoa(JSON.stringify(nftMetadata))
+  const encodedMetadata = Buffer.from(JSON.stringify(nftMetadata)).toString(
```

Comment:
thought(non-blocking): I have used `Buffer.from` in other projects in the browser. Pretty sure it should work, but we should test it anyways.